demos/python_demos/forward_tacotron_wavernn_demo/forward_tacotron_wavernn_demo.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```

Comment:
Add license headers (I'm not sure about files copied from https://github.com/keithito/tacotron. For speech_recognition_demo we have this: https://github.com/openvinotoolkit/open_model_zoo/blob/develop/demos/python_demos/speech_recognition_demo/ctcdecode-numpy/ctcdecode_numpy/ctc_beam_search_decoder.cpp).

Reply:
Done. It is MIT license.